atomic_cms.gemspec

```diff
@@ -7,6 +7,7 @@ Gem::Specification.new do |s|
   s.authors     = ['Don Humphreys']
```

Comment:
Sounds good. I do think we should contact him to see if he'll transfer ownership to us.

@gabeweaver are you able to reach out to @dhumphreys to see if he is willing to transfer the atomic_cms and atomic_assets gems to the spartan account?